project-base/app/AppKernel.php

```diff
@@ -50,11 +50,15 @@ public function registerBundles()
         ];
 
         if ($this->getEnvironment() === EnvironmentType::DEVELOPMENT) {
-            $bundles[] = new Sensio\Bundle\DistributionBundle\SensioDistributionBundle();
-            $bundles[] = new Sensio\Bundle\GeneratorBundle\SensioGeneratorBundle();
-            $bundles[] = new Symfony\Bundle\WebProfilerBundle\WebProfilerBundle();
-            $bundles[] = new Symfony\Bundle\DebugBundle\DebugBundle();
-            $bundles[] = new Symfony\Bundle\WebServerBundle\WebServerBundle();
+            $developmentBundles = [
+                new Sensio\Bundle\DistributionBundle\SensioDistributionBundle(),
+                new Sensio\Bundle\GeneratorBundle\SensioGeneratorBundle(),
+                new Symfony\Bundle\WebProfilerBundle\WebProfilerBundle(),
+                new Symfony\Bundle\DebugBundle\DebugBundle(),
+                new Symfony\Bundle\WebServerBundle\WebServerBundle(),
+            ];
+
+            $bundles = array_merge($bundles, $developmentBundles);
```

Comment:
Not sure whether this is an improvement over the previous state. Why do you prefer `array_merge`?